make.ps1

```diff
@@ -0,0 +1,138 @@
+        "filterArg" = '--where:"__FILTER__"';
+        "filters" = @{
+            "all" = "";
+            "smoke" = "Category==StandardCPython";
+            "cpython" = "Category==StandardCPython || Category==AllCPython";
+            "ironpython" = "Category==IronPython";
+        }
+    };
+    "net45" = @{
+        "runner" = "dotnet";
+        "args" = @('test', '__BASEDIR__\Src\IronPythonTest\IronPythonTest.csproj', '-f', '__FRAMEWORK__', '-o', '__BASEDIR__\bin\__CONFIGURATION__\__FRAMEWORK__', '-c', '__CONFIGURATION__', '--no-build', '-l', "trx;LogFileName=__FILTERNAME__-__FRAMEWORK__-__CONFIGURATION__-result.trx", '-s', 'runsettings.__FRAMEWORK__');
+        "filterArg" = '--filter="__FILTER__"';
+        "filters" = @{
+            "all" = "";
+            "smoke" = "TestCategory=StandardCPython";
+            "cpython" = "TestCategory=StandardCPython | TestCategory=AllCPython";
+            "ironpython" = "TestCategory=IronPython";
+        }
+    };
+    "netcoreapp2.0" = @{
+        "runner" = "dotnet";
+        "args" = @('test', '__BASEDIR__\Src\IronPythonTest\IronPythonTest.csproj', '-f', '__FRAMEWORK__', '-o', '__BASEDIR__\bin\__CONFIGURATION__\__FRAMEWORK__', '-c', '__CONFIGURATION__', '--no-build', '-l', "trx;LogFileName=__FILTERNAME__-__FRAMEWORK__-__CONFIGURATION__-result.trx", '-s', 'runsettings.__FRAMEWORK__');
+        "filterArg" = '--filter="__FILTER__"';
+        "filters" = @{
+            "all" = "";
+            "smoke" = "TestCategory=StandardCPython";
+            "cpython" = "TestCategory=StandardCPython | TestCategory=AllCPython";
+            "ironpython" = "TestCategory=IronPython";
+        }
+    }
+}
+
+function Main([String] $target, [String] $configuration) {
+    msbuild Build.proj /t:$target /p:BuildFlavour=$configuration /verbosity:minimal /nologo /p:Platform="Any CPU" /bl:build-$target.binlog
+}
+
+function Test([String] $target, [String] $configuration, [String[]] $frameworks) {
+    foreach ($framework in $frameworks) {
+        if(!$_FRAMEWORKS[$framework]["filters"].ContainsKey($target)) {
+            Write-Error "No tests available for '$target'"
+            exit 1
+        }
+
+        $filter = $_FRAMEWORKS[$framework]["filters"][$target]
+
+        $replacements = @{
+            "__FRAMEWORK__" = $framework;
+            "__CONFIGURATION__" = $configuration;
+            "__FILTERNAME__" = $target;
+            "__FILTER__" = $filter;
+            "__BASEDIR__" = $_BASEDIR;
+        };
+
+        $runner = $_FRAMEWORKS[$framework]["runner"]
+        # make a copy of the args array
+        $_TempCliXMLString = [System.Management.Automation.PSSerializer]::Serialize($_FRAMEWORKS[$framework]["args"], [int32]::MaxValue)
+        [Object[]] $args = [System.Management.Automation.PSSerializer]::Deserialize($_TempCliXMLString)
+        # replace the placeholders with actual values
+        for([int32] $i = 0; $i -lt $args.Length; $i++) {
```

Comment:
Powershell is case insensitive in its types or is int32 valid? I would have expected `int` or `Int32`

Reply:
`int32` is valid. I'll change to `int`.